apps/landing/styles.css

```
@@ -0,0 +1,427 @@
+  --indigo-hover: #4338CA;
+  --indigo-tint: #ECE9FE;
+  --accent: #4F46E5;
+  --warn: #B4530A;
+  --ok: #2E7D4F;
+  --rust: #C56A2B;
+  /* always-dark surfaces (CTA card, core layer) — inverted in light, kept dark in dark */
+  --invert-surface: #16181C;
+  --invert-ink: #F2EFE6;
+  --serif: "IBM Plex Serif", "Times New Roman", serif;
+  --sans: "IBM Plex Sans", ui-sans-serif, system-ui, -apple-system, "Segoe UI", sans-serif;
+  --mono: "JetBrains Mono", ui-monospace, "SFMono-Regular", Menlo, monospace;
```

Stylelint (17.11.1)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)